verifiers/internal/gha/provenance.go

```diff
@@ -114,6 +110,12 @@ func verifySourceURI(prov slsaprovenance.Provenance, expectedSourceURI string, v
 
 	// Last, verify that both fields match.
 	// We use the full URI to match on the tag as well.
+	if allowNoMaterialRef && len(strings.Split(materialSourceURI, "@")) == 1 {
+		// NOTE: this is an exception for npm packages built before GA,
+		// see https://github.com/gh-community/npm-provenance-private-beta-community/issues/8.
```

Comment:
It's worth noting these links aren't accessible to the general public. We probably want to create our own issue if we want to have some public info we can link to.

Reply:
Good catch. I changed the link to https://github.com/slsa-framework/slsa-verifier/issues/492